### PR TITLE
Fixes #YUPTOO-38 - Grafana Dashboard for Yuptoo

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: dashboards/grafana-dashboard-insights-yuptoo-general.configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,661 @@
+apiVersion: v1
+data:
+  dashboard.json: |-
+    {
+      "__inputs": [
+        {
+          "name": "DS_CRCS02UE1-PROMETHEUS",
```
Comment (san7ket) [inline dashboards/grafana-dashboard-insights-yuptoo-general.configmap.yaml:7]: Would this be configurable through grafana UI to switch from stage to prod ?

Comment (patilsuraj767) [reply dashboards/grafana-dashboard-insights-yuptoo-general.configmap.yaml:7]: Yes

**Comment by patilsuraj767** [comment]
Screenshot from 2022-09-20 11-40-09
This is how UI would look like. In screenshot it is showing `No data` because it is deployed on local grafana instance and not connected to prometheus datastore.

**Review by san7ket — APPROVED** [review]
ACK

**Comment by kgaikwad** [comment]
> Screenshot from 2022-09-20 11-40-09 This is how UI would look like. In screenshot it is showing `No data` because it is deployed on local grafana instance and not connected to prometheus datastore.

@patilsuraj767  - I think, you can use https://grafana.stage.devshift.net instance to verify your dashboards json with real data.

**Comment by patilsuraj767** [comment]
Here is a Yuptoo dashboard link from stage grafana - [https://grafana.stage.devshift.net/d/sGzJEgoGe/yuptoo?orgId=1&from=now-2d&to=now](https://grafana.stage.devshift.net/d/sGzJEgoGe/yuptoo?orgId=1&from=now-2d&to=now)

**Review by kgaikwad — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thank you @patilsuraj767 and @san7ket for review.

**Pull request merged**